subprojects/xcfa/xcfa/src/main/java/hu/bme/mit/theta/xcfa/model/utils/FrontendXcfaBuilder.java

```diff
@@ -790,14 +790,14 @@ public XcfaLocation visit(CWhile statement, ParamPack param) {
 		builder.addLoc(outerBeforeGuard);
 		propagateMetadata(statement, outerBeforeGuard);
 		for (int i = 0; i < (UNROLL_COUNT == 0 ? 1 : UNROLL_COUNT); ++i) {
-			if (((CCompound) body).getcStatementList().size() == 0) {
+			/*if (((CCompound) body).getcStatementList().size() == 0) {
```

Comment:
Thanks for the fix, please delete the commented lines, we have no need for them in the future.

Reply:
Commented lines removed.